nexus/src/external_api/http_entrypoints.rs

```diff
@@ -6442,14 +6442,30 @@ impl NexusExternalApi for NexusExternalApiImpl {
                 crate::context::op_context_for_external_api(&rqctx).await?;
             let params = body.into_inner();
             let system_version = params.system_version;
+
+            let current_target_release =
+                nexus.datastore().target_release_get_current(&opctx).await?;
+            let view = nexus
+                .datastore()
+                .target_release_view(&opctx, &current_target_release)
+                .await?;
+            if let views::TargetReleaseSource::SystemVersion { version } =
+                view.release_source
+            {
+                if version > system_version {
+                    return Err(HttpError::for_bad_request(
+                        None,
+                        "can't downgrade system".into(),
```

Comment:
```suggestion
                        "The requested target system release ({system_version}) is older than the current target system release ({version}).  This is not supported.".into()
```

I'm imagining an end user might see this message and want to make sure it's clear what's going on and what they've done wrong.

Reply:
Indeed, thanks! Also added in c2233b941.